roles/openshift_hosted/templates/registry_config_secret.j2

```diff
@@ -7,3 +7,6 @@ metadata:
     provider: {{ openshift.hosted.registry.storage.provider }}
 data:
   config.yml: {{ registry_config }}
+  {% if openshift_hosted_registry_storage_gcs_keyfile is defined -%}
+  gcs.json: "{{ load('file', openshift_hosted_registry_storage_gcs_keyfile) | to_nice_json }}"
+  {% endif -%}
```

Comment:
I'm pretty sure this is an orphaned file. I can see nothing that includes it. Lets get rid of it.